docs/introduction.md

```diff
@@ -261,6 +261,12 @@ for each subcommand. Each service or component is listed under the product it be
 
 None
 
+### `sat jobstat`
+
+#### PBS
```

Comment:
Is this really only provided with PBS and not Slurm?

Reply:
Correct, according to Vinay the state checker service is only found in the PBS product stream.